src/parsers/texttracks/ttml/nodes.ts

```diff
@@ -19,7 +19,8 @@
  * @returns {Element}
  */
 function getBodyNode(tt : Element) : Element|null {
-  return tt.getElementsByTagName("body")[0];
+  const bodyNodes = tt.getElementsByTagName("body");
+  return bodyNodes.length ? bodyNodes[0] : null;
```

Comment:
Minor nitpick but we usually make sure that booleans are used in all conditions and ternary operators, to avoid issues with difficult-to-predict "falsy" or "truthy" value (to replace them with much easier to predict `true` and `false` values).

So here it would be more something like:
```ts
return bodyNodes.length > 0 ? bodyNodes[0] : null; // or `>= 1` or `!== 0`... as you wish
```
Not that there's anything wrong with the current form, but that's a rule we try to stick to.

That's weird that eslint didn't catch that considering that I thought the [`strict-boolean-expressions`](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/strict-boolean-expressions.md) typescript-eslint rule covered that. I re-tested on my side but it doesn't either.

Reply:
ok done